extensions/wikia/Search/js/WikiaSearch.js

```diff
@@ -91,37 +147,38 @@ require(['search-tracking', 'uuid', 'wikia.trackingOptIn'], function(searchTrack
 			});
 
 		},
-		trackSearchResultClick: function(clickedElement) {
-			var queryparams = new URL(window.location).searchParams;
-			var query = queryparams.get('search') || queryparams.get('query');
-
-			if (!query) {
-				return;
+		trackSearchResultClick: resultClickTrackerFactory(
+			'article',
+			function (clickedElement) {
+				return clickedElement.getAttribute('data-wiki-id') + '_' + clickedElement.getAttribute('data-page-id');
+			},
+			function () {
+				return {
+					searchType: getCurrentScope()
```

Comment:
Why is this a lambda? Can current scope change dynamically? I thought it's a page refresh.

Reply:
You are right, we didn't think about it when scope was introduced. I will change that, thanks!